spec/pactas_itero/api/customers_spec.rb

```diff
@@ -160,6 +160,20 @@
     end
   end
 
+  describe ".customer" do
+    it "requests the correct resource" do
+      client = PactasItero::Client.new(bearer_token: "bearer_token")
+      request = stub_get("/api/v1/customers/customer-id").to_return(
+        body: fixture("customer.json"),
+        headers: { content_type: "application/json; charset=utf-8" },
+      )
+
+      client.customer("customer-id")
+
+      expect(request).to have_been_made
+    end
```

Comment:
Imho we should add another example to show that and how a customer is returned (similar to the example in line 52: https://github.com/shipcloud/pactas_itero/pull/11/files#diff-a2d2b9f1f86d0a222e2b0516d563d371R52). What do you think?